mappings/net/minecraft/world/gen/feature/AbstractFlowerFeature.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS net/minecraft/class_4624 net/minecraft/world/gen/feature/AbstractFlowerFeature
+	METHOD method_13175 getFlowerToPlace (Ljava/util/Random;Lnet/minecraft/class_2338;Lnet/minecraft/class_3037;)Lnet/minecraft/class_2680;
```

Comment:
Maybe `createFlower`/`createFlowerState` would be better? It's shorter and makes it clear it's the flower that will be placed, not the current flower that's there.

Reply:
hmm, it is actually an old mapping from https://github.com/FabricMC/yarn/pull/905/files#diff-68dc9155eb3a314965255406af6a0026L2. will fix